packages/dropdowns/src/elements/Fields/Message.tsx

```diff
@@ -11,6 +11,7 @@ import { Message as FormMessage } from '@zendeskgarden/react-forms';
 import { VALIDATION } from '../../utils/validation';
 
 export interface IMessageProps extends HTMLAttributes<HTMLDivElement> {
+  /** Sets the validation */
```

Comment:
Enhance the description by helping the user understand a bit more about what this does – i.e. an icon is added to the message.

See https://garden.zendesk.com/components/select#validation

Reply:
Fixed in latest push.